tests/plugins/test_sendmail.py

```diff
@@ -800,6 +800,8 @@ class WF(object):
          .with_args(True, False, False, False).and_return(True))
         flexmock(p).should_receive('_get_receivers_list').and_return(receivers)
         flexmock(p).should_receive('_fetch_log_files').and_return(None)
+        flexmock(p).should_receive('_get_image_name_and_repos').and_return(('foobar', ['baz']))
```

Comment:
This actually replaces the method from the plugin. Do we have any explicit unit test for it?